src/containers/FetchDataContainer/FetchDataContainer.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactNode } from "react";
+
+import { Loader } from "semantic-ui-react";
+
+interface FetchDataContainerProps {
+  isLoading: boolean;
+  children?: ReactNode;
+  error?: any;
+}
+
+const FetchDataContainer: FC<FetchDataContainerProps> = ({
+  isLoading,
+  children,
+  error,
+}: FetchDataContainerProps) => {
+  if (isLoading) {
+    return <Loader />;
+  }
+  if (error) {
+    // Display the error for now.
+    // TODO: throw the error and catch it with a general error boundary that displays the error page.
+    return <span>{error}</span>;
```

Comment:
It would be nice to have a task for after v3 of making this a pretty error or something... Just like providing something for the user to look at besides a span.

Reply:
I expect we'll make a general Error component that pops a modal or something.